test/smoketest/main.py

```diff
@@ -560,7 +570,7 @@ def county_audit(ac, county_id):
 
     r = test_endpoint_get(ac, ac.state_s, "/dos-dashboard")
 
-    r = test_endpoint_json(ac, county_s, "/intermediate-audit-report", {})
+#    r = test_endpoint_json(ac, county_s, "/intermediate-audit-report", {})
```

Comment:
After you commit this, I'll switch the intermediate reporting to a periodic thing, and/or, when my flexible-periodic-config code lands, make it part of that.